toplevel/coqargs.ml

```diff
@@ -416,7 +420,22 @@ let parse_args ~help ~init arglist : t * string list =
       Flags.profile_ltac_cutoff := get_float opt (next ());
       oval
 
-    |"-require" -> add_vo_require oval (next ()) None (Some false)
+    |"-rfrom" ->
+      let from = next () in add_vo_require oval (next ()) (Some from) None
+
+    |"-require" ->
+      warn_deprecated_simple_require ();
+      add_vo_require oval (next ()) None (Some false)
+
+    |"-require-import" | "-ri" -> add_vo_require oval (next ()) None (Some false)
```

Comment:
Maybe that's a good moment to turn `(Some false)` into `~scope:Import` ?

Reply:
We could, but it is quite a lot of changes, up to `library.ml`. So, maybe later.

Reply:
Ok then.